tests/network/vmi_multus.go

```diff
@@ -1243,6 +1246,9 @@ var _ = SIGDescribe("[Serial]Macvtap", func() {
 					return "", err
 				}
 
+				if vmiIP == "" {
+					return "", fmt.Errorf("no IP detected on interface with mac %s", macAddress)
+				}
```

Comment:
can this happen ?
if timeout results in err, then its already covered on line 1245
https://github.com/kubevirt/kubevirt/pull/6280/files#diff-da14ca97e77db6bb275427ac69187a539c58fc297e64519628e5385f9f411c5fR1245

Reply:
Hmmm... Yes, you are right.
Before it did not timeout, but now it should. I will adjust it.

Reply:
@oshoval HI. If you meant to ask if it's possible that the `vmiIP` is empty, then yes. We've observed this behavior here for example: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/directory/pull-kubevirt-e2e-k8s-1.20-sig-network/1428719305444298752#1:build-log.txt%3A6651

Also here:
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/periodic-kubevirt-e2e-k8s-1.19-sig-network/1425381817116004352#1:build-log.txt%3A6441
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/periodic-kubevirt-e2e-k8s-1.19-sig-network/1425502317083365376#1:build-log.txt%3A6338

Actually I think we always hit the empty IP string when this test fails on connectivity check before the migration

Reply:
Done, I just removed it.
I will wait to see if the tests now passes and then will push.